### DISCUSSION TITLE
Data Management Zone deployment asks for existing Enterprise Scale Landing Zone

### DISCUSSION THREAD

**Discussion by fabianjogschies** [Azure/data-management-zone#115] [General] [answered]
Is it a recommendation to deploy management zone first or a must do?

![image](https://user-images.githubusercontent.com/6885215/127496510-761c263c-39c7-452c-8e2c-402cc7144dfe.png)

if i deploy a management zone it asks me if i already got a landing zone.

Doc points out its a must do:
**https://github.com/Azure/data-landing-zone/blob/main/docs/EnterpriseScaleAnalytics-Prerequisites.md#prerequisites**

**Comment by marvinbuss** [comment, marked as answer]
Sorry for the late reply @fabianjogschies. This is not asking you for the Data Landing Zone, but whether you deploy this into an existing Enterprise-Scale Landing Zone environment. With this we mean this reference architecture: https://github.com/Azure/Enterprise-Scale

How can we make this more clear?
Would you like us to change the wording here?

**Reply by marvinbuss** [reply]
The link provided in the blade is also helping users. Should I change that?

This is the [current link](https://docs.microsoft.com/en-us/azure/cloud-adoption-framework/ready/enterprise-scale/architecture#high-level-architecture).

**Reply by marvinbuss** [reply]
@fabianjogschies Do you want us to rename the option above? I want to make sure, that there is no misunderstanding for users. What is your recommendation?

**Reply by fabianjogschies** [reply]
Hi Marvin,

i'm not quiet sure how ta handle. If you know what different types of Zone's are available it's clear. Only the word "Landing" can be a little confusing if you read it for the first time.
Since the naming convention is already done as Enterprise Scale Landing Zone i would leave it as it is. And maybe add a hint before the learn more link with "what types of zones are available?"

**Reply by marvinbuss** [reply]
Let me think about this today. I will make some changes to make this more clear. Thanks for the feedback!
